master/usage/configuration/index.md

```diff
@@ -1,11 +1,7 @@
 ---
-title: Configuring Calico
+title: Configuring Systems for use with Calico
```

Comment:
Not sure what should be done with this doc yet.  

It's kind of just a jumble of random bits of information that might be useful, mostly Openstack related.

@neiljerram does it make sense to move the remaining contents of this doc to one of the Openstack specific parts of the docs?